routes/user.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const router = express.Router();
+const { User } = require('../db');
+
+function create(req, res, next) {
+   return User.create(req.body)
+    .then(user => res.send(user))
+    .catch(err => next(err));
+  }
+
+function list(req, res, next) {
+  return User.find()
+    .then(users => res.send(users))
+    .catch(err => next(err));
+};
+
+function find(req, res, next) {
+  const id = req.params.id;
+
+  return User.findById(id)
+    .then(user => res.send(user))
+    .catch(err => next(err));
+};
+
+function update(req, res, next) {
+  return User.findByIdAndUpdate(req.params.id, req.body)
+    .then(user => res.send(user))
+    .catch(err => next(err));
+};
+
+function remove(req, res, next) {
+  return User.findByIdAndRemove(req.params.id)
+    .then(user => res.send(user))
+    .catch(err => next(err));
+};
```

Comment:
Since we don't want to send users password to client ever create an array of private data keys, such as 'password'.
And before you send `user` object omit those keys. You can use [lodash omit](https://lodash.com/docs/4.17.4#omit).
You can also make a separate helper function called sth like `sanitizeUser` which will take user, omit private data and return it.